src/platform/loader-web.ts

```diff
@@ -78,7 +78,7 @@ export class PlatformLoader extends PlatformLoaderBase {
       error(x);
     }
     // clean up
-    delete self.defineParticle;
+    delete self['defineParticle'];
```

Comment:
This looks suspicious to me as it makes the loader temporarily invalidate it's type. I think it's okay because the as the change is at run time, it shouldn't hit any type checks, though I'm not entirely sure about that.

Reply:
agree, this also feels a bit brittle.  @sjmiles might have some ideas on how to address this?